### DISCUSSION TITLE
traceback.txt issue on my Chromebook

### DISCUSSION THREAD

**Discussion by Funtime-Foxy-Rules** [Monika-After-Story/MonikaModDev#9600] [QA - tech support + general] [answered]
I start up the game and I added in all the files to the game file and all that but it says there's an issue so please help. 😅
![Screenshot 2022-08-01 10 21 03 PM](https://user-images.githubusercontent.com/110438868/182278644-e5f7e1bb-8698-497e-b8ea-05cf7d7e8df3.png)
![Screenshot 2022-08-01 10 21 24 PM](https://user-images.githubusercontent.com/110438868/182278655-06e33b0b-f79d-461f-a47f-ca54060b093d.png)
![Screenshot 2022-08-01 10 21 44 PM](https://user-images.githubusercontent.com/110438868/182278668-09d5345c-7722-4df6-8425-7569dd23a455.png)

**Comment by Wingdinggaster656** [comment, marked as answer]
It's very likely you didn't install the game correctly. Do it once more.
